src/introspection/getSchemaFromData.spec.js

```diff
@@ -104,7 +104,7 @@ const QueryType = new GraphQLObjectType({
 */
 
 test('creates one type per data type', () => {
-    const typeMap = getSchemaFromData(data).getTypeMap();
+    const typeMap = getSchemaFromData(data, {}).getTypeMap();
```

Comment:
`getSchemaromData` should have a default value for options to avoid that kind of change.